packages/atomic/src/pages/examples/headless.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html dir="ltr" lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0" />
+    <title>Coveo Atomic: Headless example</title>
+
+    <script type="module" src="/build/atomic.esm.js"></script>
+    <script nomodule src="/build/atomic.js"></script>
+    <link rel="stylesheet" href="/themes/coveo.css" />
+    <script type="module">
+      import {loadAdvancedSearchQueryActions, loadContextActions} from '/build/headless/headless.esm.js';
+      // CDN example: 'https://static.cloud.coveo.com/atomic/v1/atomic.esm.js/headless/headless.esm.js'
+
+      function setExpression(engine) {
+        const action = loadAdvancedSearchQueryActions(engine).updateAdvancedSearchQueries({
+          aq: '@author=amoreau',
+        });
+        engine.dispatch(action);
+      }
+
+      function setContext(engine) {
+        const action = loadContextActions(engine).addContext({
+          contextKey: 'headless',
+          contextValue: 'very good',
```

Comment:
😂 

While I personally find it funny, I think we should just add something more generic, like `userGroup: 'sales'` or something like that.

Reply:
yeah I had no idea what to put here